test/chpldoc/compflags/author/CATFILES

```diff
@@ -0,0 +1 @@
+docs/modules/author.doc.html
```

Comment:
We try to reserve all caps files for general operations that should be performed on any tests that might be added to the directory.  The CLEANFILES you've added here fits that category, but I think the other three are more test-specific, so should be applied to just the test itself